test/Microsoft.SqlTools.ManagedBatchParser.LocalizationTests/SrTests.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Globalization;
+using Xunit;
+using Microsoft.SqlTools.ServiceLayer.BatchParser.ExecutionEngineCode;
+using ManagedBatchParserSr = Microsoft.SqlTools.ManagedBatchParser.SR;
+namespace Microsoft.SqlTools.ManagedBatchParser.LocalizationTests
```

Comment:
Could you please rename this project `Microsoft.SqlTools.ManagedBatchParser.UnitTests` or something like that so it is a bit more generic?  I think having an entire project for a singles localization test is a bit overkill, so if you do need a new test project it should be able to contain all batch parser tests.

As a side note, this test doesn't look particularly useful, and I think it's just for code coverage purposes.  If it doesn't impact the code coverage numbers, you might consider removing this test.